<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.258">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Luciano V. Cosme">

<title>Luciano’s website - Research</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Luciano’s website</span>
    </a>
  </div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html">
 <span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./about.html">
 <span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./research.html" aria-current="page">
 <span class="menu-text">Research</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./publications.html">
 <span class="menu-text">Resume</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./pomeranians.html">
 <span class="menu-text">Pomeranians</span></a>
  </li>  
</ul>
              <div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#research-overview-unraveling-the-genetics-of-disease-carrying-mosquitoes" id="toc-research-overview-unraveling-the-genetics-of-disease-carrying-mosquitoes" class="nav-link active" data-scroll-target="#research-overview-unraveling-the-genetics-of-disease-carrying-mosquitoes">Research Overview: Unraveling the Genetics of Disease-Carrying Mosquitoes</a>
  <ul class="collapse">
  <li><a href="#project-1-population-genomics-of-globally-distributed-arbovirus-vectors-aedes-albopictus-and-aedes-aegypti" id="toc-project-1-population-genomics-of-globally-distributed-arbovirus-vectors-aedes-albopictus-and-aedes-aegypti" class="nav-link" data-scroll-target="#project-1-population-genomics-of-globally-distributed-arbovirus-vectors-aedes-albopictus-and-aedes-aegypti"><strong>Project 1: “Population Genomics of Globally Distributed Arbovirus Vectors, Aedes albopictus and Aedes aegypti”</strong></a></li>
  <li><a href="#project-2.-decoding-mosquito-attraction-through-gene-and-microrna-analysis" id="toc-project-2.-decoding-mosquito-attraction-through-gene-and-microrna-analysis" class="nav-link" data-scroll-target="#project-2.-decoding-mosquito-attraction-through-gene-and-microrna-analysis">Project 2. Decoding Mosquito Attraction through Gene and microRNA Analysis</a></li>
  <li><a href="#project-3.-balancing-pest-control-and-protecting-beneficial-insects" id="toc-project-3.-balancing-pest-control-and-protecting-beneficial-insects" class="nav-link" data-scroll-target="#project-3.-balancing-pest-control-and-protecting-beneficial-insects">Project 3. Balancing Pest Control and Protecting Beneficial Insects</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Research</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Luciano V. Cosme </p>
          </div>
  </div>
    
  
    
  </div>
  

</header>

<section id="research-overview-unraveling-the-genetics-of-disease-carrying-mosquitoes" class="level1">
<h1>Research Overview: Unraveling the Genetics of Disease-Carrying Mosquitoes</h1>
<section id="project-1-population-genomics-of-globally-distributed-arbovirus-vectors-aedes-albopictus-and-aedes-aegypti" class="level2">
<h2 class="anchored" data-anchor-id="project-1-population-genomics-of-globally-distributed-arbovirus-vectors-aedes-albopictus-and-aedes-aegypti"><strong>Project 1: “Population Genomics of Globally Distributed Arbovirus Vectors, Aedes albopictus and Aedes aegypti”</strong></h2>
<p><strong>Duration:</strong> August 2015 – Present</p>
<p><strong>My Journey in Vector Genomics:</strong> Since August 2015, my research has concentrated on the genetic underpinnings of two globally significant arbovirus vectors: <em>Aedes albopictus</em> and <em>Aedes aegypti</em>. These mosquitoes are not just pests but pivotal in transmitting critical diseases like dengue fever, substantially threatening global public health.</p>
<p><strong>Fundamental Discoveries and Publications:</strong> My journey in this domain has led to several notable publications in esteemed scientific journals. Some key highlights include:</p>
<ul>
<li><strong>Genotyping Array for <em>Ae. albopictus</em>:</strong> In a groundbreaking study published in Parasites &amp; Vectors (2023), I developed a novel genotyping array, a crucial tool for studying the genetic makeup of the invasive <em>Ae. albopictus</em> mosquito. Below is a figure from the chip development manuscript, where I analysed the population structure of <em>Ae. albopictus</em> at its native range.</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="admixture_r2_0.1_MANUSCRIPT.jpg" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">Population structure of <em>Ae. albopictus</em> in the native range using 57,780 SNPs. Panel A shows the results of Admixture analyses on 237 samples for K=5. Each bar represents a mosquito. The country of origin is listed on the bottom, while the population code and the name of the sampling locality are on the top. Each bar color and height represent the probability of the mosquito being assigned to an ancestral group. On the y-axis are the admixture proportions for each individual. Panel B shows a map where the ancestry matrix is interpolated over the entire region from which samples with different shades of the same color reflect different ancestry coefficients. The map was obtained using the R package tess3r. The colored pies at each sampling site reflect the proportion of the clusters found at that site.</figcaption><p></p>
</figure>
</div>
<ul>
<li><strong>Genome-Wide Association Studies (GWAS) on <em>Ae. aegypti</em>:</strong> My research, featured in Frontiers in Genetics (2022), revealed new genetic loci associated with pyrethroid resistance in <em>Ae. aegypti</em>, offering new insights into combating insecticide resistance. Below is a figure from this study showing a PCA, a quantile quantile and Manhattan plots.</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="gwas_kdr.jpg" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">GWAS in mosquitoes with different levels of knockdown resistance. (A) Principal component analysis showing two different clusters; (B) Quantile-quantile plot with confidence intervals; (C) Manhattan plot with the SNPs with the lowest p values annotated per chromosome. Green dots are SNPs on the vgsc gene. The SNP chip has 11 SNPs on the vgsc gene; only 3 SNPs passed our filtering, with most being removed due to low minor allele frequency.</figcaption><p></p>
</figure>
</div>
<p>However, my favorite part of the study was to analyse the genome architecture difference between resistant and susceptible mosquitoes. First. I looked what other SNPs may be linked to SNPs identified in the GWAS:</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="ld_snp1.jpg" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">Linkage block analysis of loci AX-93253438 on <em>Ae. aegypti</em> chromosome 2 using the R package LDBlockShow. (A) Manhattan plot with significance line (red). SNPs are colored following the key at top right. (B) Genes in the region with SNPs marked as green lines, where the CDS is in yellow, introns are in light blue, UTR is in pink, and intergenic regions are orange. (C) Heatmap with D’ estimates.</figcaption><p></p>
</figure>
</div>
<p>Next, I did linkage network analysis. This part was fascinating since I discovered different linkage patterns between the resistant and susceptible mosquitoes in two independent populations, indicating assortative matting, where susceptible females avoid matting with resistant males, perhaps because their wing beat might be slower due to the fitness cost of the <em>kdr</em> mutations.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="ld_network.jpg" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">A snapshot of the entire networks at an LD threshold value just above that any single outlier clusters (SOCs) merge for resistant and susceptible individuals within each population. Each SOC is shown at an LD threshold where it is joined by a single link to other loci, in increasing order of threshold from right to left for each group. The compound outliers are not shown.</figcaption><p></p>
</figure>
</div>
<p>We can recreate the mosaic like linkage patterns for each chromosome:</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="linkage_chr.jpg" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">LD clusters bigger than 1 Mb in susceptible and resistant individuals within each population.</figcaption><p></p>
</figure>
</div>
<ul>
<li><strong>Evolution of <em>kdr</em> Haplotypes in <em>Ae. aegypti</em>:</strong> Published in PLOS Neglected Tropical Diseases (2020), this study illuminated the independent origins of the F1534C kdr mutation, enhancing our understanding of the evolutionary dynamics of insecticide resistance.</li>
</ul>
<p>Here is an overview of the sodium channel and know <em>kdr</em> mutations:</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="sodium_channel.jpeg" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">Scheme of the voltage gated sodium channel. The IIS6 and IIIS6 segments are highlighted, indicating the primers (arrows showing orientation) used to amplify their corresponding DNA regions in this study. The <em>kdr</em> mutation sites previously found in Aedes aegypti in these segments (989, 1011, 1016, 1532 and 1534) are also indicated.</figcaption><p></p>
</figure>
</div>
<p>Here is the summary of the frequencies of all the haplotypes I found:</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="kdr_frequencies.jpg" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">Frequencies of haplotypes with or without non-synonymous substitutions in IIS6 and IIIS6 NaV segments of <em>Ae. aegypti</em> worldwide populations. The populations contain one bar for each of the two <em>NaV</em> segments. In each bar it is presented the frequency of the haplotypes with non-synonymous SNPs and the sum of the remaining haplotypes (wild-type). Populations are grouped according to their continent.</figcaption><p></p>
</figure>
</div>
<p>I found evidence of two independent origins of the 1534C <em>kdr</em> mutation in <em>Ae. aegypti</em>.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="kdr_origins.jpg" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">Evidence for at least two independent origins for the 1534C <em>kdr</em> mutation in <em>Ae. aegypti</em>. Bars show the haplotype frequencies of IIS6 and IIIS6 <em>NaV</em> segments found in five localities that are fixed for 3s6-01 haplotype (A). The IIS6 haplotypes from Clades A or B are represented as purple and green lines, respectively. Two different origins are proposed for the 1534C kdr mutation in B and C panels. The single origin for the other <em>kdr</em> mutations is also suggested in panel C. Phased haplotypes are followed by the name of the populations where they were found. <em>Kdr</em> SNPs are shown in red.</figcaption><p></p>
</figure>
</div>
<p><strong>Technical Summary:</strong> My approach integrates advanced genomic tools with field studies. By employing SNP chips and GWAS, I have identified crucial genomic regions associated with resistance to dengue infection and pyrethroid detoxification in mosquitoes. This research has been pivotal in understanding the geographic variations in genetic factors affecting disease transmission rates.</p>
<p><strong>Significance and Impact:</strong> The significance of this project extends beyond academic realms into crucial public health domains. By uncovering the genetic mechanisms behind mosquito survival, disease transmission, and insecticide resistance, my work lays the foundation for more effective mosquito management strategies. This is particularly pertinent in the United States, where controlling the spread of dengue fever is an escalating public health concern.</p>
</section>
<section id="project-2.-decoding-mosquito-attraction-through-gene-and-microrna-analysis" class="level2">
<h2 class="anchored" data-anchor-id="project-2.-decoding-mosquito-attraction-through-gene-and-microrna-analysis">Project 2. Decoding Mosquito Attraction through Gene and microRNA Analysis</h2>
<p><strong>Understanding Mosquito Attraction:</strong> In this pivotal project, I delve into the molecular mechanisms underlying mosquito attraction to humans. The central focus is on two key species: <em>Anopheles coluzzii</em>, a known malaria transmitter, and <em>Ae. aegypti</em>, the primary carrier of dengue fever. I aimed to unravel the role of olfaction genes in determining host preference, a crucial factor in disease transmission.</p>
<p><strong>Research Methodology:</strong> My approach involved extracting RNA from the antennae of these mosquitoes, where I targeted two primary types of molecules: messenger RNA (mRNA) and microRNAs. These are instrumental in regulating gene expression. Utilizing next-generation sequencing (NGS) technologies, I captured a comprehensive snapshot of active genes, focusing on those that play a role in olfaction.</p>
<p><strong>Key Insights and Publications:</strong></p>
<ul>
<li><p><strong>Chemosensory Gene Expression:</strong> My research, published in Parasites &amp; Vectors (2020) and BMC Genomics (2017), highlighted the differences in chemosensory gene expression between male and female mosquitoes, shedding light on potential genes that influence host preference.</p></li>
<li><p><strong>MicroRNA Research in Vector Control:</strong> Presentations at various scientific meetings, such as the Annual Meeting of the ASTMH and the Society for Vector Ecology, emphasized the potential of microRNAs in developing transgenic mosquito control strategies.</p></li>
</ul>
<p><strong>Bioinformatics Analysis:</strong> Employing tools like EdgeR and DESeq2 within the R programming environment, I analyzed the differential expression of olfactory genes. This analysis revealed which genes were more active in female mosquitoes, potentially influencing their attraction to humans.</p>
<p><strong>Implications of the Research:</strong> This project is not just an academic endeavor but a step towards tangible public health solutions. We can develop more effective mosquito control strategies by understanding the genetic underpinnings of mosquito attraction to humans. This could include new repellents or attractants, directly impacting the transmission of diseases like malaria and dengue fever.</p>
<p><strong>Significance in the US Context:</strong> With mosquito-borne diseases growing in regions like Florida and Puerto Rico, this research is particularly relevant to the United States. The insights gained can lead to improved repellents and control measures, safeguarding public health and potentially reducing healthcare costs associated with mosquito-borne illnesses.</p>
<p><strong>Aligning with Broader Goals:</strong> This project aligns seamlessly with my broader goal of mitigating the impact of mosquito-borne diseases through advanced genetic research. The findings contribute significantly to our understanding of the transmission cycle of pathogens and aid in developing novel disease control strategies.</p>
</section>
<section id="project-3.-balancing-pest-control-and-protecting-beneficial-insects" class="level2">
<h2 class="anchored" data-anchor-id="project-3.-balancing-pest-control-and-protecting-beneficial-insects">Project 3. Balancing Pest Control and Protecting Beneficial Insects</h2>
<p>Impact of Commonly Used Insecticides on Natural Enemies</p>
<p><strong>Exploring Insecticide Effects on Beneficial Insects:</strong> In this research, I have delved into the effects of various insecticides on beneficial insects, crucial for natural pest control. My studies have focused on understanding the delicate balance between effective pest management and preserving these beneficial species.</p>
<p><strong>Key Findings and Publications:</strong></p>
<ul>
<li><p><strong>Sub-Lethal Effects of Permethrin:</strong> In a study published in the Journal of Economic Entomology (2009), I explored the reaction of a generalist predator to permethrin, revealing a sub-lethal stimulatory response that suggests complex regulatory mechanisms.</p></li>
<li><p><strong>Neem Oil and Lacewing <em>Chrysoperla externa</em>:</strong> My research in Arquivos do Instituto Biológico (2009) highlighted the significant impact of neem oil on the development stages of the lacewing, an essential natural pest controller.</p></li>
<li><p><strong>Botanical and Synthetic Insecticides on Lady Beetles:</strong> The study in Arquivos do Instituto Biológico (2007) brought attention to the potential risks posed by insecticides to the lady beetle <em>Cycloneda sanguinea</em>, a vital predator in controlled environments.</p></li>
</ul>
<p><strong>Field Studies and Broader Implications:</strong> My field studies have included evaluating the dynamics of <em>C. externa</em> in various agricultural systems, emphasizing sustainable practices. The toxicity analysis of various pesticides on this natural predator in coffee crops (Neotropical Entomology, 2005) and citrus crops (Neotropical Entomology, 2004) underlines the importance of integrated pest management strategies.</p>
<p><strong>Significance of the Research:</strong> This project is fundamental for informing agricultural professionals about using insecticides responsibly and preserving beneficial insects integral to natural pest control. By understanding this balance, we can maintain healthy crop ecosystems and reduce reliance on chemical pest control methods. This leads to more sustainable agricultural practices, which are crucial for ecosystems and food supply.</p>
<p><strong>Relevance to the United States:</strong> In the context of the United States, this research supports sustainable agriculture and pest management strategies vital for the nation’s food supply and environmental health. Understanding how to protect beneficial insects allows for natural pest population reduction, minimizing chemical use, and lowering production costs.</p>
<p><strong>Aligning with Broader Goals:</strong> This research complements my genomics work by promoting sustainable environmental practices essential for the long-term effectiveness of vector control programs. By identifying less harmful insecticides to beneficial insects, my work supports maintaining natural pest control mechanisms, reducing reliance on chemical interventions contributing to resistance.</p>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>